nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -90,6 +90,8 @@ BtNavigator::on_configure(const rclcpp_lifecycle::State & /*state*/)
   blackboard_->set<rclcpp::Node::SharedPtr>("node", client_node_);  // NOLINT
   blackboard_->set<std::shared_ptr<tf2_ros::Buffer>>("tf_buffer", tf_);  // NOLINT
   blackboard_->set<std::chrono::milliseconds>("node_loop_timeout", std::chrono::milliseconds(10));  // NOLINT
+  blackboard_->set<std::chrono::milliseconds>("send_goal_timeout", std::chrono::milliseconds(1000));  // NOLINT
```

Comment:
Is 1 second enough time for this timeout?

Reply:
I think any value would be arbitrary in the end (just like the `node_loop_timeout` of 10 milliseconds), so perhaps they both could be a run-time parameter? As a default, 1 second seemed sufficient, though I can't say how long you'd want to wait before returning that the request failed.